docs/data/rpc/admin-guide.mdx

```diff
@@ -481,31 +549,28 @@ STELLAR_CORE_TIMEOUT = "2s"
 # fields in the config toml from being parsed.
 # STRICT = false
 
-# configures the transaction retention window expressed in number of ledgers,
-# the default value is 1440 which corresponds to about 2 hours of history
-TRANSACTION_RETENTION_WINDOW = 1440
 ```
 
 Note that the above generated configuration contains the default values and you need to substitute them with proper values to run the image.
 
 ### Build From Source
 
-Instructions for building Stellar RPC from source can be found [here](https://github.com/stellar/soroban-rpc/blob/main/cmd/soroban-rpc/README.md).
+Instructions for building Stellar RPC from source can be found [here](https://github.com/stellar/stellar-rpc/blob/main/cmd/stellar-rpc/README.md).
 
-## Use the RPC Instance
+## Use your Local RPC Instance
 
-You can configure Stellar CLI to use a remote RPC endpoint:
+You can configure Stellar CLI to use your local remote RPC endpoint:
 
 ```bash
-stellar network add --global testnet \
-    --rpc-url https://soroban-testnet.stellar.org:443 \
+stellar network add --global local \
+    --rpc-url http://localhost:8000 \
     --network-passphrase 'Test SDF Network ; September 2015'
```

Comment:
the network passphrase will change when using the `local` network. It should be `Standalone Network ; February 2017` i believe. double-check by looking a `local` container's Horizon endpoint `http://localhost:8000` the network passphrase is reported there.

Reply:
Opps nice catch thanks!  Verified that is the correct passphrase by checking quickstart:  https://github.com/stellar/quickstart/blob/master/README.md#--local